Mention 2022.09.1 hotfix in SH changelog (#2909)
gitpod-io/website: src/lib/contents/changelog/2022-10-07.md

This release includes security fixes addressing information leakage in logs; see the [security announcement log](https://www.gitpod.io/security/log) for more information.

### 2022.09.1 hotfix (published on 14.7.2022)

> [#13821](https://github.com/gitpod-io/gitpod/pull/13821) This hotfix includes a change to allow `protected_secrets` (see [security announcement log](https://www.gitpod.io/security/log) for more information) to work when using self-signed certs.

Reviewer: Just a nit 😬 

```suggestion
[#13821](https://github.com/gitpod-io/gitpod/pull/13821) This hotfix includes a change to fix the regression caused by `protected_secrets` (see [security announcement log](https://www.gitpod.io/security/log) for more information) when working with self-signed certs.
```
